.circleci/config.yml

```diff
@@ -84,6 +84,31 @@ references:
       path: test-reports
       destination: test-reports
 
+  benchmark: &benchmark
+    run:
+      name: Run benchmarks
+      command: |
+          pip install --user gitpython
+          echo 'export PYTHONPATH=$PYTHONPATH:.' >> $BASH_ENV
+          source $BASH_ENV
+          BENCHSCR="benchmarks/benchmarker.py"
+          echo "Running benchmark on current HEAD"
+          python "$BENCHSCR" run -c
+          echo "Comparing with orign/master"
+          git reset --hard origin/master
+          echo "PYVER=${PYVER}"
+          rm freud/*.cpp
+          python${PYVER} setup.py build_ext --inplace --COVERAGE --ENABLE-CYTHON
+          git checkout "${CIRCLE_SHA1}" -- benchmarks/
+          echo "Running benchmark on origin/master"
+          python "$BENCHSCR" run -c
+          python "$BENCHSCR" compare origin/master "${CIRCLE_SHA1}"
```

Comment:
I'd really like to put the comparison into its own task so that it appears as a separate section in the CircleCI output. Something called `Compare benchmarks`, right below `Run benchmarks`.

![image](https://user-images.githubusercontent.com/3943761/57939119-9a8f7b00-7897-11e9-8354-31a10ea4318f.png)

Reply:
This would also clarify that the benchmarks succeeded but their _comparison_ failed (e.g. by being slow).